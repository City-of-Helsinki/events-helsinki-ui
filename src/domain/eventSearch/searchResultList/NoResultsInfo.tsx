import { IconSearch } from 'hds-react';
import React from 'react';
import { useTranslation } from 'react-i18next';

import styles from './noResultsInfo.module.scss';

const NoResultsInfo: React.FC = () => {
  const { t } = useTranslation();

  return (
    <div className={styles.noResultsInfo}>
      <div className={styles.iconWrapper}>
        <IconSearch aria-hidden/>
      </div>
      <div className={styles.bigText}>
        {t('eventSearch.noResultsInfo.bigText')}
      </div>
      <div className={styles.smallText}>
        {t('eventSearch.noResultsInfo.smallText')}
      </div>
    </div>
  );
};

export default NoResultsInfo;
