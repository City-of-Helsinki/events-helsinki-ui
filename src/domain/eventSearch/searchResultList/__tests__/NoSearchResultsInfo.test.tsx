import { render } from '@testing-library/react';
import * as React from 'react';

import NoResultsInfo from '../NoResultsInfo';

test('matches snapshot', () => {
  const { container } = render(<NoResultsInfo />);

  expect(container.firstChild).toMatchSnapshot();
});
