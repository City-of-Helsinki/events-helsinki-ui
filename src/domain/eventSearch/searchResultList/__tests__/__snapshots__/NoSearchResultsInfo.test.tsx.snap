// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  class="noResultsInfo"
>
  <div
    class="iconWrapper"
  >
    <svg
      class="Icon-module_icon__1Jtzj icon_hds-icon__1YqNC Icon-module_s__2WGWe icon_hds-icon--size-s__2Lkik"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        fill="none"
        fill-rule="evenodd"
      >
        <path
          d="M0 0h24v24H0z"
        />
        <path
          d="M15 1a8 8 0 11-4.798 14.402l-6.401 6.4-1.591-1.59 6.398-6.4A8 8 0 0115 1zm0 2a6 6 0 100 12 6 6 0 000-12z"
          fill="currentColor"
        />
      </g>
    </svg>
  </div>
  <div
    class="bigText"
  >
    Voi vitsi, valitsemillasi hakuehdoilla ei löytynyt yhtään hakutulosta
  </div>
  <div
    class="smallText"
  >
    Kokeile muuttamalla hakuehtoja ja varmasti löydät jotain kiinnostavaa tekemistä!
  </div>
</div>
`;
